### PR TITLE
[WIP] Awesome API docs

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

**Comment by diasdavid** [comment]
It is my impression, or Coveralls forgot that we deactivated them all and now comments in all PRs?

**Comment by dignifiedquire** [comment]
@diasdavid no we just haven't disabled them everywhere yet

**Comment by harlantwood** [comment]
Nice.  One nit: http://libp2p.github.io/js-peer-id/#Introduction should probably link back to https://github.com/libp2p/js-peer-id , in case someone lands there from search or something...

File: src/index.js [hunk 1]
```diff
@@ -1,31 +1,61 @@
...
+   * The private key of this id, if it exists.
+   *
+   * @type {RSAPrivateKey|undefined}
+   */
   get privKey () {
     return this._privKey
   }
 
+  /**
+   * The private key of this id, if it exists.
```
Comment (VictorBjelkholm) [inline src/index.js]: "The public key of this id"

**Comment by dignifiedquire** [comment]
@harlantwood just updated and added a github link: http://libp2p.github.io/js-peer-id/

**Comment by vasco-santos** [comment]
Closing this as we already have the API docs in the README and these are outdated

**Pull request closed without merging**